### PR TITLE
docs: added vee-validate plugin custom labels documentation

### PR DESCRIPTION
**Pull request by logaretm**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other: documentation update

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing: https://github.com/formvuelate/formvuelate/blob/main/.github/CONTRIBUTING.md#development-setup
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

This PR documents the fixed behavior in https://github.com/formvuelate/formvuelate-plugin-vee-validate/pull/20 where it introduces custom field labels.

### CODE REVIEW

File: docs/3.x/src/guide/veevalidate.md [hunk 1]
```diff
@@ -284,6 +284,62 @@ export default {
 </script>
 ```
 
+## Validation Messages Labels <Badge text="2.3.1" type="warning" vertical="middle" />
```
Comment (logaretm) [inline docs/3.x/src/guide/veevalidate.md]: @marina-mosti I probably need to fix this, which version are we tagging for that release?

EDIT: NVM

File: docs/3.x/src/guide/veevalidate.md [hunk 2]
```diff
@@ -284,6 +284,62 @@ export default {
 </script>
 ```
 
+## Validation Messages Labels <Badge text="2.4.0" type="warning" vertical="middle" />
+
+::: warning Important
+This section doesn't apply to validation schemas created with `yup`. You can use `yup` own [`.label` method](https://github.com/jquense/yup#mixedlabellabel-string-schema) to provide friendly display names for your fields.
```
Comment (marina-mosti) [inline docs/3.x/src/guide/veevalidate.md]:
```suggestion
This section doesn't apply to validation schemas created with `yup`. You can use `yup`'s own [`.label` method](https://github.com/jquense/yup#mixedlabellabel-string-schema) to provide friendly display names for your fields.
```

**Review by marina-mosti — APPROVED** [review]

File: docs/3.x/src/guide/veevalidate.md (hunk 2 above)
Comment (logaretm) [reply docs/3.x/src/guide/veevalidate.md]: Thanks 🤦‍♂️

**Comment by marina-mosti** [comment]
Thanks @logaretm merging!

**Pull request merged**